### ISSUE TITLE
Temporal language in ‘Effect’ definition

### ISSUE THREAD

**Issue by gregfowlerphd** [CommonCoreOntology/CommonCoreOntologies#474]
Why does the textual definition employ the temporal locutions ‘follows’ and ‘previous’? Such locutions aren’t present in the case of ‘Cause’. Moreover, they aren’t reflected in the logical (equivalence class) definition. Thus, changing the textual definition of ‘Effect’ to

> A Process that is caused by some other Process.

would bring it more in line with both the ‘Cause’ definition and with the logical definition.

**Comment by swartik** [comment]
@gregfowlerphd I suspect the reason the definition uses "follows" is because Princeton's WordNet does. Unfortunately the URL for the definition source isn't responding so I can't verify my suspicion.

Actually I rather like the temporal constraint that the causing process starts earlier. (I don't think you can express that in Description Logics, though.) I'd rather change the definition of Cause. Possible counter-argument: a desire for CCO to handle quantum entanglement.

**Comment by gregfowlerphd** [comment]
@swartik: Yeah, I'm fine with the temporal language staying, so long as the two definitions are parallel. Though I do think that if we keep it, the equivalence axiom should be replaced by a subclass axiom (since it clearly won't say the same thing as the textual definition if the temporal language is retained).

Hope some other people weigh in on their preferred way of making the definitions parallel!

**Comment by neilotte** [comment]
@gregfowlerphd IMHO, cause and effect should be removed from CCO entirely. In particular, as there are different notions of causation and as there is also the matter of proximity in analyzing causes, it's unclear how to apply these classes generally and different domain ontologies are likely to perform this identification in different ways that do not need to be encouraged by the appearance of this classes in CCO. 

I would be interested to know how much value folks using CCO find in these classes vs a strategy where they create their own local notions of cause and effect relative to application-specific criteria for identification. In my experience, the analysis of cause in science is usually done by performing statistical analyses once the data has been organized by an ontology, rather than represented per se by an ontology. @johnbeve @mark-jensen @APCox  -- thoughts?
